fixtures/sample-project-with-build-process/package.json

```diff
@@ -20,5 +20,16 @@
     "import": "yo liferay-fragments:import",
     "import:watch": "yo liferay-fragments:import --watch",
     "preview": "yo liferay-fragments:preview"
+  },
+  "dependencies": {
+    "react": "16.8.6",
+    "react-dom": "16.8.6"
+  },
+  "devDependencies": {
```

Comment:
Aren't these unneeded? :thinking: They should be provided by the fragments CLI...